examples/gsd/_01_gsd_iluz.py

```diff
@@ -90,7 +89,7 @@ def load_matlab_output(datapoint):
 short_trial_matlab_output = load_matlab_output(short_trial)
 short_trial_reference_parameters = short_trial.reference_parameters_.wb_list
 
-short_trial_output = GsdIluz().detect(short_trial.data_ss, sampling_rate_hz=short_trial.sampling_rate_hz)
```

Comment:
Can you change this back to `data_ss` (and everywhere else this might have been changed)?